pages/api/content/fr/typescript.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+  title: 'TypeScript',
+  subtitle: 'Utilisation des définitions TypeScript de `glamorous`',
+  description: `
+    Les définitions TypeScript actuelles sont incomplètes et reposent sur les besoins des développeurs qui ont contribué. 
+    
+    Les pull requests pour les améliorer sont les bienvenues et appréciées. Si vous n'avez jamais contribué à l'open source, vous pouvez trouver cette [vidéo gratuite](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) qui est très utile.
+    
+    ## Support complet
+
+    ### Config
+    - [x] useDisplayNameInClassName
+
+    ### Styles dynamiques
+
+    Pour utiliser des styles dynamiques avec des props personnalisées, utilisez des génériques. Exemple:
+    
+    ~~~javascript
+    const MyStyledDiv = glamorous.div<{noPadding?: boolean}>(
+      {
+        margin: 1,
+      },
+      props => ({
+        padding: props.noPadding ? 0 : 4,
+      })
+    )
+
+    <MyStyledDiv /> // styles appliqués : {margin: 1, padding: 4}
+    <MyStyledDiv noPadding /> // styles appliqués : {margin: 1, padding: 0}
+    ~~~
+
+    ## Support incomplet
+
+    ### Sécurité de propriété CSS
+
+    * les pseudo-classes
+    * les pseudo-éléments
+    * les sélecteurs de CSS relationnels
+    * Les Media Queries
+
+    Tous ceux-là fonctionnent, cependant, vous obtenez seulement une sûreté du typage et intellisense sur les simples clés ccs des props (voir le [typage css](https://github.com/paypal/glamorous/blob/master/typings/css-properties.d.ts)).
```

Comment:
> "Toutes celles-là" (les propriétés)

OK, i fix it.

> "clés CSS des props"

OK, i fix it.

> "une sûreté du typage et intellisense [...]" = il ne manque pas quelque chose ?

I thought intellisense had a relationship with IntelliSense. But maybe I'm wrong? What is the exact meaning of intellisense in the English version? cc @kentcdodds @williamolojede

If I'm right, then I could change the translation into:
`vous obtenez seulement une sûreté du typage et la capacité d'utiliser intelliSense`